docs/developer/available-plugins/adyen.mdx

```diff
@@ -62,26 +61,25 @@ The query returns all required fields to prepare a payment form with methods ava
     }
   }
 }
- ```
+```
 
 ## Creating a payment in Saleor
 
-To create a payment properly, you need to follow the steps described in 
-[Creating a checkout session](developer/checkout.mdx#creating-a-checkout-session) and 
+To create a payment properly, you need to follow the steps described in
+[Creating a checkout session](developer/checkout.mdx#creating-a-checkout-session) and
 [Selecting a shipping method](developer/checkout.mdx#selecting-a-shipping-method).
 
-The next step is to choose Adyen as the payment gateway that we want to use to process the checkout. 
+The next step is to choose Adyen as the payment gateway that we want to use to process the checkout.
 For the Adyen payment gateway, we need to provide:
 
 - `checkoutId`: ID of the checkout, for which new payment should be created.
 
-- `input.gateway`: ID of the payment gateway which should be assigned to this payment. For Adyen it will be:  `mirumee.payments.adyen`.
+- `input.gateway`: ID of the payment gateway which should be assigned to this payment. For Adyen it will be: `mirumee.payments.adyen`.
```

Comment:
```suggestion
- `input.gateway`: ID of the payment gateway which should be assigned to this payment. For Adyen, it will be: `mirumee.payments.adyen`.
```